vector/src/main/java/im/vector/app/features/VectorFeatures.kt

```diff
@@ -28,6 +28,7 @@ interface VectorFeatures {
     fun tchapIsVoipSupported(): Boolean
     fun tchapIsCrossSigningEnabled(): Boolean
     fun tchapIsKeyBackupEnabled(): Boolean
+    fun tchapIsThreadEnabled(): Boolean
```

Comment:
I think you don't need to add `tchapIsThreadEnabled()` function. We have `areThreadMessagesEnabled` function for that. The last one depend on `settings_labs_thread_messages_default` value.